tests/blend.py

```diff
@@ -257,86 +254,32 @@ def test_multi_channel(channel_check, channel_check_inverse, range_all,
     np.testing.assert_array_equal(expected, result)
 
 
-def test_channel_size_mismatch(range_all, color_white):
+def test_linear_rgb_size_mismatch(range_all, color_white):
     '''Test supplying channels with different dimensions'''
 
     input_channels = [
         {
-            'image': np.uint16([0]),
+            'image': np.array([[0, 0, 0]], dtype=np.uint16),
             'color': color_white,
             'min': range_all[0],
             'max': range_all[1]
         },
         {
-            'image': np.uint16([[0, 65535]]),
+            'image': np.array([[0, 65535]], dtype=np.uint16),
             'color': color_white,
             'min': range_all[0],
             'max': range_all[1]
         }
     ]
 
     with pytest.raises(ValueError,
-                       match=r'All channel images must have equal dimensions'):
-        linear_bgr(input_channels)
+                       match=r'.*broadcast.*'):
+        linear_rgb(input_channels)
 
 
-def test_channel_size_zero():
+def test_linear_rgb_size_zero():
     '''Test supplying no channels'''
 
     with pytest.raises(ValueError,
                        match=r'At least one channel must be specified'):
```

Comment:
No need for string match.

Reply:
done in `refactor names and revert to version 0.0.1`

Reply:
https://github.com/sorgerlab/minerva-lib-python/pull/17/commits/29ed61d369e2d506ea15686cd45fd49e12580ee1#diff-4438daa1ceea9cdeeef65ab8cfb5075dR282